packages/manager/modules/octavia-load-balancer/src/create/constants.js

```diff
@@ -46,6 +46,41 @@ export const REGION_AVAILABILITY_LINK = {
   DEFAULT: 'https://www.ovhcloud.com/en/public-cloud/regions-availability/',
 };
 
+export const GETTING_STARTED_LINK = {
+  FR:
+    'https://docs.ovh.com/fr/load-balancer-octavia/premiers-pas-avec-load-balancer-public-cloud/',
+  DE:
+    'https://docs.ovh.com/de/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  ES:
+    'https://docs.ovh.com/es/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  IE:
+    'https://docs.ovh.com/ie/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  IT:
+    'https://docs.ovh.com/it/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  PL:
+    'https://docs.ovh.com/pl/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  PT:
+    'https://docs.ovh.com/pt/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  GB:
+    'https://docs.ovh.com/gb/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  CA:
+    'https://docs.ovh.com/ca/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  QC:
+    'https://docs.ovh.com/ca/fr/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  WS:
+    'https://docs.ovh.com/us/es/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  AU:
+    'https://docs.ovh.com/au/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  SG:
+    'https://docs.ovh.com/sg/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  ASIA:
+    'https://docs.ovh.com/asia/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
+  US:
+    'https://docs.ovh.com/us/en/publiccloud/network-services/getting-started-with-load-balancer-public-cloud/',
```

Comment:
```suggestion
  US:
    'https://support.us.ovhcloud.com/hc/en-us/articles/18610207964051-Getting-Started-with-Load-Balancer-on-Public-Cloud',
```
US are one dedicated support guides site.

For other, i think is more efficient put the new url that start by https://help.ovhcloud.com/csm/